examples/express/src/components/MyFragment.jsx

```diff
@@ -0,0 +1,34 @@
+// components/MyFragment.js
```

Comment:
To remove? Or:
```suggestion
// components/MyFragment.jsx
```

By the way, maybe could we use TypeScript also in the examples.

Reply:
I'll remove it, it comes from the example
> By the way, maybe could we use TypeScript also in the examples.
Yes, I'll do that